docs/login-testcase.md

```diff
@@ -0,0 +1,21 @@
+# Login testcase
+
+## Prerequisites
+User has acount in the system already created by system admin.
+
+## Steps
+
+### Step 1
+User opens pah-fm website.
+
+### Expected results 1
+User sees login view with 2 text fields: "Username" and "Password" and 1 button "Login".
+User sees 3 flages: polish, english, ukrainian, in the right side, at top of the page.
+
+### Step 2
+User enters his email (attached to already created account) into "Username" field.
+User enters his password (attached to already created account) into "Password" field.
+User clicks "Login" button.
+
+### Expected results 2
+User succesfully logs in and sees main view of application.
```

Comment:
Instead of main view I would say - is redirected to homepage (`/`). "Main view" might be misleading.